### PR TITLE
Add mockup pattern development support in buildout

### PR DESCRIPTION
**Pull request by kakshay21**
see #212

### CODE REVIEW

File: bobtemplates/plone/buildout/buildout.cfg.bob [hunk 1]
```diff
@@ -42,4 +42,15 @@ recipe = zc.recipe.testrunner
...
-    ${buildout:eggs}
\ No newline at end of file
+    ${buildout:eggs}
+
+[zopepy]
+recipe = zc.recipe.egg
+eggs =
+   Products.CMFPlone
+   ${instance:eggs}
+   ${test:eggs}
```
Comment (jensens) [inline bobtemplates/plone/buildout/buildout.cfg.bob]: I don't think we need the test-eggs here

File: bobtemplates/plone/buildout/buildout.cfg.bob [hunk 2]
```diff
@@ -42,4 +42,15 @@ recipe = zc.recipe.testrunner
 defaults = ['--auto-color', '--auto-progress']
 eggs =
     Plone
-    ${buildout:eggs}
\ No newline at end of file
+    ${buildout:eggs}
+
+[zopepy]
```
Comment (jensens) [inline bobtemplates/plone/buildout/buildout.cfg.bob]: In order to reduce confusion, I would name either name the section ``plone-helper-scripts`` or similar.

Comment (MrTango) [reply bobtemplates/plone/buildout/buildout.cfg.bob]: regarding the zopepy it is useful for using autocompletion in editors like sublime text or vs code.
If you set the Python interpreter for the editor, your editor will have all buildout packages and do autocompletion. @jensens if you know a better generic way to do archive this, i'm happy to here it.

File: bobtemplates/plone/buildout/buildout.cfg.bob (hunk 1 above)
Comment (MrTango) [reply bobtemplates/plone/buildout/buildout.cfg.bob]: for autocompletion test-eggs meight be usefull too, so i would keep it in here.

File: CHANGES.rst [hunk 3]
```diff
@@ -4,7 +4,8 @@ Changelog
 3.3.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add mockup pattern development support in buildout
```
Comment (MrTango) [inline CHANGES.rst]: Please descibe what you actually did, not what the title of the issue was.
Think about what yu PR will add and describe it.

File: bobtemplates/plone/buildout/buildout.cfg.bob (hunk 2 above)
Comment (MrTango) [reply bobtemplates/plone/buildout/buildout.cfg.bob]: naming the part: plone-helper-scripts is a good idea, let's do it.

File: bobtemplates/plone/buildout/buildout.cfg.bob (hunk 1 above)
Comment (kakshay21) [reply bobtemplates/plone/buildout/buildout.cfg.bob]: Do we need test-eggs then? @jensens @MrTango

**Pull request merged**